### PR TITLE
fix: [GIT-533]: Update the navigation from enterprise gitops to gitops

### PR DESCRIPTION
**Pull request by GeorgiosP**
##### Summary:

Updating the navigation to no longer say enterprise GitOps and just GitOps 

##### Jira Links:
[GIT-533](https://harness.atlassian.net/browse/GIT-533)

##### Screenshots:

Screen Shot 2022-01-24 at 11 57 12 AM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -263,7 +263,8 @@
     "@types/testing-library__react": "^10.0.0",
     "@types/testing-library__dom": "^7.0.0",
     "anser": "2.0.1",
-    "create-react-context": "0.3.0"
+    "create-react-context": "0.3.0",
+    "webpack": "5.65.0"
```
Comment (GeorgiosP) [inline package.json:267]: We had multiple versions of webpack causing some build issues. I pinned and resolved them to the version we use `5.65.0`

**Comment by GeorgiosP** [comment]
retrigger title-check

**Comment by GeorgiosP** [comment]
retrigger Title_Check

**Comment by GeorgiosP** [comment]
retrigger Cypress

**Comment by GeorgiosP** [comment]
retrigger cypress

**Comment by GeorgiosP** [comment]
retrigger jest

**Comment by GeorgiosP** [comment]
@vkbansal when you have some time can you give this a look over, I was told to ping you on review for this thanks!

**Comment by GeorgiosP** [comment]
retrigger typecheck

**Comment by GeorgiosP** [comment]
retrigger cypress

**Comment by GeorgiosP** [comment]
retrigger cypress

**Comment by GeorgiosP** [comment]
retrigger Cypress

**Comment by GeorgiosP** [comment]
retrigger cypress

**Review by vkbansal — APPROVED** [review]

**Review by cjlee01 — APPROVED** [review]

**Pull request merged**